content/docs/studio/troubleshooting.md

```diff
@@ -147,6 +148,20 @@ view. Once you show/hide columns, you can save the changes. Check out the
 [above video](#showhide-columns) to see how you can show/hide columns. Once you
 show/hide columns, remember to save the changes.
 
+## View does not contain some of my commits or branches
+
+This is likely not an error. DVC Studio identifies commits that do not change
+metrics and will auto-hide such non-experiment commits. You can also manually
```

Comment:
We don't [explain experiments](https://dvc.org/doc/user-guide/experiment-management/experiments-overview) as commits. In fact technically they are not commits but custom Git references. So hopefully there's no need for this clarification.

```suggestion
metrics and will auto-hide such commits. You can also manually
```

Reply:
Is this a common phrase in Studio docs though?

Reply:
No, it's not. We just started using this term for commits that are likely not experimentation attempts. Eg, if someone added a readme file, or if they restructured their repo, or did anything else that is not about updating params, files, metrics etc., then all those commits are non-experiments. I used the same term in the last Studio product update email. But if you have a suggestion for a different way of writing this, do suggest.

Reply:
If we change this here, we also need to change it in `explore-experiments.md`; coz there too, I've used this term. Does it make sense to call them `noisy` commits or `noise`? Or better to stick with just `.. such commits`?

Reply:
> Eg, if someone ... restructured their repo, or did anything else that is not about updating params, files, metrics etc., then all those commits are non-experiments

Oh. I see. So we're not even talking about our special "DVC Experiments" (`dvc exp run`) here. This can be confusing, let me think 🤔

Reply:
yes, this isn't about `exp`. I think it's okay to just call them `such commits`, meaning, commits that don't change metrics and files.

Reply:
I think we should just go with

"commits where metrics did not change and will hide them automatically"

Don't mix the concept of experimentation here 🙂

Reply:
Yes, I removed term `non-experiments`.